blockly-core/core/code_generation/generator.js

```diff
@@ -197,6 +197,11 @@ Blockly.CodeGenerator.prototype.blockToCode = function(block, opt_showHidden) {
   if (!block) {
     return '';
   }
+  if (block.skipGeneration) {
+    // Clear the flag, so it doesn't persist to future serializations.
+    block.skipGeneration = false;
```

Comment:
what about a "skipNextGeneration" flag that is permanently applied to the `studio_ask` block, and which this method checks against before generating the code for the next block?